packages/ckeditor5-clipboard/src/dragdrop.ts

```diff
@@ -345,7 +345,7 @@ export default class DragDrop extends Plugin {
 
 			const { clientX, clientY } = ( data as DomEventData<DragEvent> ).domEvent;
 
-			dragDropTarget.updateDropMarker( data.target, data.targetRanges, clientX, clientY, this._blockMode );
+			dragDropTarget.updateDropMarker( data.target, data.targetRanges, clientX, clientY, this._blockMode, this._draggedRange );
```

Comment:
This line could look more readable, see my comment in `dragdroptarget.ts`.

Reply:
This would be a breaking change because this method is public and this PR will land in the next release (after the one where we'll introduce the `DragDropTarget` class).

Also, to be consistent, we should also change the params in the `getFinalDropRange` method, which would introduce a lot of noise in the PR.

Unless this is a hard blocker, I would skip this change.

Reply:
See my response in other comment.